openwisp_monitoring/monitoring/tests/test_models.py

```diff
@@ -77,11 +91,18 @@ def test_write(self):
         self.assertEqual(measurement['value'], 2)
 
     def test_general_write(self):
-        m = self._create_general_metric(name='Sync test')
+        # TODO: Remove or modify this test
+        m = self._create_general_metric(name='Sync test', key='Sync test')
         m.write(1)
         measurement = list(query('select * from sync_test').get_points())[0]
         self.assertEqual(measurement['value'], 1)
 
+    def test_metric_write_wrong_related_fields(self):
+        m = self._create_general_metric(name='ping', configuration='ping')
+        extra_values = {'reachable': 0, 'rtt_avg': 0.51, 'rtt_max': 0.6, 'rtt_min': 0.4}
+        with self.assertRaises(ValueError):
```

Comment:
It would assert also that  the exception message contains 'reachable not defined in metric configuration'